Sources/Rendering/Core/ColorTransferFunction/index.js

```diff
@@ -389,13 +389,17 @@ function vtkColorTransferFunction(publicAPI, model) {
       // todo
       const idx = publicAPI.getAnnotatedValueIndexInternal(x);
       if (idx < 0 || numNodes === 0) {
-        publicAPI.getNanColor(rgb);
```

Comment:
It would be a convenient signature to also support in `macros.js!getArray()` function

Reply:
I certainly think it's a good way to write an array type directly into a destination array. It'll involve rewriting a lot of typescript definitions to include the optional out param...